sqlstore/src/main/resources/db/migration/V8123__SequencerPartitionContainer.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE SequencerPartitionContainer ADD COLUMN sequencingParametersId bigint,
+ADD CONSTRAINT `fk_sequencerPartitionContainer_sequencingParameters` FOREIGN KEY
+(sequencingParametersId) REFERENCES SequencingParameters
+(parametersId);
+ALTER TABLE SequencingParameters ADD COLUMN movieTime int UNSIGNED;
```

Comment:
Right now, if anyone attempts to save any of the existing "Custom (see notes)" sequencing parameters for PacBio, they'll get an error because no movie time is set. Is there a default value that would make sense to apply to existing sequencing parameters?